### PR TITLE
Update api sha to release0.8 latest.

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -1301,7 +1301,7 @@
     "rbac/v1alpha1",
     "routing/v1alpha1"
   ]
-  revision = "170445541e5b5a1515b52c9c9596570220679a27"
+  revision = "4eac2c3d96dcbe2b56006e6bdfa643d160695da8"
```
Comment (rkpagadala) [inline Gopkg.lock]: maybe we should change the branch in line 1289 to release-0.8

Comment (qiwzhang) [reply Gopkg.lock]: Done

Comment (rkpagadala) [reply Gopkg.lock]: Thanks. I forgot to mention earlier. Please add to Gopkg.toml

**Pull request merged**